rust-toolchain.toml

```diff
@@ -1,3 +1,4 @@
 [toolchain]
 profile = "default"
 channel = "1.78.0"
+targets = ["x86_64-apple-ios", "aarch64-apple-ios", "aarch64-apple-ios-sim", "aarch64-linux-android", "armv7-linux-androideabi", "x86_64-linux-android", "i686-linux-android"]
```

Comment:
Don't we need a desktop targets for linux and macos as well?

Reply:
If you want, it makes it easier to set up cross compilation on new machines as you don't have to run the `rustup install target ...` commands but this is entirely optional

Reply:
This force the installation of too many toolchains that are useless most of the time, and it just broke my installation by creating conflicts.

Reply:
Feel free to remove it

Reply:
How did it create conflicts? It seems pretty harmless except for using some disk space, no?